scss/components/pagination.scss

```diff
@@ -31,12 +33,14 @@ $block: #{$fd-namespace}-pagination;
     list-style: none;
     margin-bottom: 0;
   }
-  &__link {
+  &__link {  @include fd-reset();
```

Comment:
Minor nit: `@include fd-reset()` should be on the next line.

Reply:
done